README.md

```diff
@@ -16,7 +16,7 @@ An extension for Gnome-Shell to show Persian date and calendar
 * Go to the cloned directory
 * run ```make``` command in the terminal
 * Restart Gnome-shell. (ALT+F2, r, Enter)
-* You may need to enable the extension via Gnome Extensions Application
+* You may need to enable the extension via ```Gnome Extensions Application```
```

Comment:
We can use one single backtick here instead of three.
And also I think it can be Gnome `Extensions` Application.

Reply:
Thank you for your comment. I resolved this and made the backticks consistent throughout the README.